### PR TITLE
Add run_long_tests parameter and use it in uyuni

### PR DESCRIPTION
**Pull request by ktsamis**
Add run_long_tests parameter and use it in uyuni, if it works fine, I will port it to the other environments.

### CODE REVIEW

**Comment by ktsamis** [comment]
I'm not sure the second export is needed in the secondary stage since I define it before. But just to be safe.

File: jenkins_pipelines/environments/common/pipeline.groovy [hunk 1]
```diff
@@ -30,16 +30,16 @@ def run(params) {
...
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS_ENABLED=${params.run_long_tests}; cd /root/spacewalk/testsuite; rake cucumber:reposync'"
             }
             stage('Core - Initialize clients') {
                 sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:init_clients'"
             }
             stage('Secondary features') {
-                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
+                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS_ENABLED=${params.run_long_tests}; cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
```
Comment (srbarrios) [inline jenkins_pipelines/environments/common/pipeline.groovy:42]: I'm not sure neither. Just play in your console, see if the value stay or it's only there during the command.

**Review by srbarrios — APPROVED** [review]

File: jenkins_pipelines/environments/common/pipeline.groovy (hunk 1 above)
Comment (cbosdo) [reply jenkins_pipelines/environments/common/pipeline.groovy:42]: in the Uyuni PR introducing this feature, the env variable is `LONG_TESTS`, the export needs to be matching it

Comment (ktsamis) [reply jenkins_pipelines/environments/common/pipeline.groovy]: Renamed to `long_tests` to match, I will try locally to see if the secondary is needed.

Comment (ktsamis) [reply jenkins_pipelines/environments/common/pipeline.groovy]: I don't think the env variables are persistent. From what I see in the current run of uyuni at least. But I will check again when we merge this.

**Comment by cbosdo** [comment]
The PR adding the `LONG_TESTS` variable has now been merged. Feel free to merge this one then

**Comment by srbarrios** [comment]
We can't merge, I still see this wrong:
```
export LONG_TESTS_ENABLED=${params.long_tests};
```

It must be what Cedric suggested:
```
export LONG_TESTS=${params.long_tests};
```

**Comment by srbarrios** [comment]
Now is good, squash and merging ;)

**Pull request merged**